osquery/database/tests/database_tests.cpp

```diff
@@ -78,6 +134,24 @@ TEST_F(DatabaseTests, test_delete_values) {
   EXPECT_TRUE(value.empty());
 }
 
+TEST_F(DatabaseTests, test_delete_values_int) {
+  int expected = 0;
+  setDatabaseValue(kLogs, "k", 0);
+
+  int value;
+  getDatabaseValue(kLogs, "k", value);
+  EXPECT_EQ(value, expected);
+
+  auto s = deleteDatabaseValue(kLogs, "k");
+  EXPECT_TRUE(s.ok());
+
+  // Make sure the key has been deleted.
+  value = -5;
+  s = getDatabaseValue(kLogs, "k", value);
+  EXPECT_FALSE(s.ok());
+  EXPECT_EQ(value, -5);
+}
+
```

Comment:
IMHO, it is good also to test writing to DB max acceptable integer numbers here, like `std::numeric_limits<int>::max()`. And to test writing to DB string values with symbols inside (`@!#$%?|/.,:\\^&*([])` and so on).